src/Features/CSharp/Portable/InvokeDelegateWithConditionalAccess/InvokeDelegateWithConditionalAccessAnalyzer.cs

```diff
@@ -92,8 +92,7 @@ private void SyntaxNodeAction(SyntaxNodeAnalysisContext syntaxContext)
             var expressionStatement = (ExpressionStatementSyntax)innerStatement;
 
             // Check that it's of the form: "if (a != null) { a(); }
-            var invocationExpression = ((ExpressionStatementSyntax)innerStatement).Expression as InvocationExpressionSyntax;
-            if (invocationExpression == null)
+            if (!(((ExpressionStatementSyntax)innerStatement).Expression is InvocationExpressionSyntax invocationExpression))
```

Comment:
Can't we use `expressionStatement` from above and simplify to `expressionStatement.Expression is InvocationExpressionSyntax invocationExpression`

Reply:
indeed.

Reply:
done.